### ISSUE TITLE
Where is Meta-target loss

### ISSUE THREAD

**Issue by LearnerHong** [medtray/DAME#4]
Thank you very much for your last reply, and I would like to ask you another question. Your paper mentions the use of meta-learning in the training process, but I didn't see it when I read your source code. Can you point out where your code uses Meta-target loss?

**Comment by medtray** [comment]
Thank you for your question. In the training phase of domain adaptation, we have K domains, and in order to simulate the testing scenario, we consider K-1 domains as the meta-sources and the remaining source domain as the meta-target. So, these are the terminologies used in domain adaptation. In line 335 and 336 of model.py, if it is the training phase, we remove the features that correspond to the groundtruth label (considered as meta-target), in order to obtain the features of the remaining source domains (meat-sources), and we apply attentions and compute logits of the meta-sources in line 366 of model.py. Then, the meta-target loss is computed in line 389.

 Here is a paper to have more understanding about the meta-sources and meta-target (Guo et al. 'Multi-Source Domain Adaptation with Mixture of Experts').

**Comment by LearnerHong** [comment]
Thank you for your patience!
